test/integration_test.go

```diff
@@ -7522,3 +7523,198 @@ func (ts *IntegrationTestSuite) TestRawValueQueryMetadata() {
 	ts.Equal("Basic", metadata.Definition.Type)
 	ts.Equal(3, len(metadata.Definition.QueryDefinitions))
 }
+
+func (ts *IntegrationTestSuite) TestWorkflowTaskFailureMetric_BenignHandling() {
+	wfWithApplicationErr := func(ctx workflow.Context, isBenign bool) error {
+		if !isBenign {
+			return temporal.NewApplicationError("Non-benign failure", "", false, nil)
+		}
+		return temporal.NewApplicationErrorWithOptions(
+			"Benign failure",
+			"",
+			temporal.ApplicationErrorOptions{
+				Category: internal.ErrorCategoryBenign,
```

Comment:
test should not refer to `internal` types, you need to export `ErrorCategoryBenign`

Reply:
Exported/exposed in common error.